src/components/AccountContainer.js

```diff
@@ -44,17 +44,39 @@ class AccountContainer extends Component {
         }
       ]
     }
+    this.handleChange = this.handleChange.bind(this)
   }
 
-  handleChange(event) {
-    // your code here
+  fetchData(){
+    fetch('https://boiling-brook-94902.herokuapp.com/transactions')
+    .then(res => res.json())
+    .then(transactions => this.setState({transactions}))
+  }
+
+  componentDidMount(){
+    this.fetchData()
   }
 
+  handleChange(event) {
+    const query = this.state.searchTerm
```

Comment:
I think what you're trying to do here is filter your list, but it seems like you're not sure what to do with it. We discussed this in the review, but the way I would do this is to filter the list here and then just pass it down into the `TransactionList` component